src/pages/ValidateLoginNewWorkspacePage.js

```diff
@@ -20,13 +20,17 @@ const propTypes = {
 
     /** The accountID and validateCode are passed via the URL */
     route: validateLinkPropTypes,
+
+    /** List of betas */
+    betas: PropTypes.arrayOf(PropTypes.string),
 };
 
 const defaultProps = {
     route: {
         params: {},
     },
     session: {},
+    betas: null,
```

Comment:
Similar to other comment - I think we should default this to `[]`

Reply:
Left a response [here](https://github.com/Expensify/App/pull/4205#discussion_r676750815) and the same applies here. Let me know if you feel differently.

Reply:
Agreed that this is the same as above - no comments needed here 😆